main.go

```diff
@@ -23,23 +25,32 @@ func main() {
 	)
 	flag.Parse()
 
-	opts := providerserver.ServeOpts{
-		Address: "registry.terraform.io/HPE/hpe",
-		Debug:   debug,
-	}
-
 	p := provider.New(
 		version,
 		morpheus.New(),
-		// subprovider2.New(),
```

Comment:
It would be good to preserve showing how additional 
subproviders would be added (however it would be done)